sdk/src/assertion.rs

```diff
@@ -251,8 +253,8 @@ impl Assertion {
     // }
 
     // Return version string of known assertion if available
-    pub(crate) fn get_ver(&self) -> Option<usize> {
-        self.version
+    pub(crate) fn get_ver(&self) -> usize {
+        self.version.unwrap_or(1)
```

Comment:
Nitpick: Maybe make the "default" or "fallback" version a constant to check against?